fulp_modules/features/toys/code/plushies.dm

```diff
@@ -87,3 +87,21 @@
 	icon = 'fulp_modules/features/toys/icons/toys.dmi'
 	icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
+
+/obj/item/toy/plush/beefplushie/living_beefplushie
+	desc = "It looks oddly alive. You feel like you should pet it."
+	COOLDOWN_DECLARE(beefplushie_cooldown)
+	var/beefplushie_cooldown_time = 2 MINUTES
```

Comment:
This should be a define

Reply:
how do i turn it into a define

Reply:
new to this, just for my info. What difference does this make?